### DISCUSSION TITLE
Exhausted available authentication methods

### DISCUSSION THREAD

**Discussion by eeb74** [iterate-ch/cyberduck#14574] [Q&A]
Hi, I am having the same issue as posted in #8463 & #8486 and have not been able to resolve it by removing my old password credentials from keychain and reinstalling cyberduck. I am using Cyberduck version 8.5.7 on Mac. What else can I try to troubleshoot this issue? 
<img width="778" alt="Cyberduck_troubleshooting" src="https://user-images.githubusercontent.com/113538044/234075995-0ca55346-cbeb-4e6e-816d-a90354e015bd.png">

**Comment by dkocher** [comment]
You might find #14364 helpful.
